### ISSUE TITLE
Disable "edit" action on sendouts not modfiyable

### ISSUE THREAD

**Issue by kringkaste** [putyourlightson/craft-campaign#328]
I know this is an edge case, but it would be nice if there was a similar HTML trigger for the _edit_ action as there is for _cancel_ action in sendouts. Because the slideout for edit gets a redirect with 302 from the `prepareEditScreen` function but as ajax request it doesn't work and you get an empty slideout and a server failure :)

**Comment by bencroker** [comment]
I'm not quite following, can you please explain in simpler terms?

**Comment by kringkaste** [comment]
Sorry :(

Craft 4.2.4: If you go to sendouts and click on the title of a sendout already send, everything works fine. You get a 302 redirect to the preview URL. If you select the checkbox of the sendout and click on the "edit" action, the edit panel is opening and will be empty and you get a "server failure" flash notice. So it would be better to disable the edit action if you select a sendout not modifiable. Like you do it for the cancel action.

**Comment by bencroker** [comment]
Yes, you're absolutely right, thanks for reporting this!!

**Comment by bencroker** [comment]
Fixed in https://github.com/putyourlightson/craft-campaign/commit/2d713a8bfd7cdbd5c4ef6331e4b03d70fed476e7 for the next release.

**Comment by bencroker** [comment]
Released in version 2.1.7.
